R/getWebServiceData.R

```diff
@@ -137,8 +137,8 @@ default_ua <- function() {
 
   ua <- paste0(names(versions), "/", versions, collapse = " ")
 
-  if ("UA.dataRetrieval" %in% names(options)) {
-    ua <- paste0(ua, "/", options()[["UA.dataRetrieval"]])
+  if (Sys.getenv("CUSTOM_DR_UA") != "") {
```

Comment:
Another idea that we could potentially put off for later: what if we set the environment variable on package load to some random alphanumeric string if it isn't already set elsewhere? Then we'd be able to at least track requests originating from the same R session. Perhaps something like the following:

```
if (Sys.getenv("CUSTOM_DR_UA") == "") {
  Sys.setenv("CUSTOM_DR_UA" = paste0(sample(c(LETTERS, 0:9), size = 12, replace = TRUE), collapse = ""))
}
```

Reply:
My gut says we start with advertising the change in NEWS and documentation. Then we can point a few power users to that and ask them to add it to their pipelines. THEN after you've had a chance to see how that helps (or not!) in your analytic work, we can think about adding this sort of session info. I'm worried it would just overwhelm the server log work.